react-accordion/my-app/src/TopicCard.tsx

```diff
@@ -0,0 +1,24 @@
+import './TopicCard.css';
+
+type Topic = {
```

Comment:
It's not good to have two definitions of `Topic`. Write it just once, then export and import it. If there are two and you later change your mind about how it should look, you will most likely mess up and introduce bugs.